### DISCUSSION TITLE
Frigate + Coral USB Ubuntu 24.04.1

### DISCUSSION THREAD

**Discussion by corradz** [blakeblackshear/frigate#14829] [Ask A Question]
Hello,
currently I am running frigate 0.14 using docker on a minipc with a Coral AI cpu connected via USB.
Everything is working fine, but I would like to upgrade Ubuntu on the minipc from version 22.04.5 to 24.04.1.
Before upgrading the minipc, are there any issues to run frigate on docker with the Coral USB cpu with Ubuntu 24.04.1? 
Does the Coral usb cpu work fine?

**Comment by banthungprong** [comment]
I am using a Mini-PC (Minis Forum i7) with 24.04 since July without any problems (with two Coral TPU USB).

**Reply by corradz** [reply]
Thanks a lot banthungprong!

**Comment by jubjubrsx** [comment]
not sure if its just me, but installing the coral drivers with 24.04 I ran into issues with the python version being to new (3.12) so might keep that in mind.
